### PR TITLE
feat(editing): process remove action events

### LINKED ISSUE #49: Remove action functionality
XML editing action for removing a node from a given document

### PR DESCRIPTION
**Pull request by ca-d**
Closes #49 .

### CODE REVIEW

File: test/unit/Editing.test.ts [hunk 1]
```diff
@@ -43,4 +43,11 @@ describe('Editing Element', () => {
     expect(element).to.have.attribute('name', 'A2');
     expect(element).to.not.have.attribute('desc');
   });
+
+  it('removes an element on Remove Action', () => {
+    editor.dispatchEvent(newOpenDocEvent(doc, 'test.scd'));
+    const node = doc.querySelector('Substation')!;
+    editor.dispatchEvent(newActionEvent({ node }));
+    expect(doc.querySelector('Substation')).to.not.exist;
```
Comment (JakobVogelsang) [inline test/unit/Editing.test.ts:51]: This test is also not fail save. Someone could rewrite that the function does remove the whole XMLDocument and we would not see from the test. How about testing that the parent still exists. And maybe that only the one node from the parent has been removed?

**Review by JakobVogelsang — CHANGES_REQUESTED** [review]
I think we need more tests on that. What would you say?

File: test/unit/Editing.test.ts (hunk 1 above)
Comment (ca-d) [reply test/unit/Editing.test.ts:58]: Maybe we can use https://www.npmjs.com/package/@open-wc/semantic-dom-diff to compare expected dom to actual dom?

Comment (ca-d) [reply test/unit/Editing.test.ts:58]: Also, let's not go for "fail-safe" tests documenting current behaviour after the fact, but instead use tests to specify behaviour piece-meal. This other type of test I would rather see as a regression test for an actually introduced bug.

I've been trying to develop this in an actually test driven way, something along these lines: https://www.youtube.com/watch?v=EZ05e7EMOLM

**Review by ca-d — COMMENTED** [review]
What happened to the commit history on this branch? I'm really confused. Could this still be cleaned up? Thanks!

**Comment by JakobVogelsang** [comment]
@ca-d This can be closed isn't it?

**Comment by ca-d** [comment]
> @ca-d This can be closed isn't it?

Yes, I'm closing this in favor of #61 , where I've transported these changes over to the new open-wc recommendation based project infrastructure. Thanks @juancho0202 for putting this together in preparation, I just copied the relevant parts of the code over and it worked.

**Pull request closed without merging**